mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -129,27 +132,30 @@ def ocn_modelvsobs(config, field, streamMap=None, variableMap=None):
             "{}/MODEL.SST.HAD187001-198110.OI198111-201203.nc".format(
                     observationsDirectory)
         dsObs = xr.open_mfdataset(obsFileName)
-        # Select years for averaging (pre-industrial or present-day)
-        # This seems fragile as definitions can change
-        if yearOffset < 1900:
-            timeStart = datetime.datetime(1870, 1, 1)
-            timeEnd = datetime.datetime(1900, 12, 31)
-            preindustrialText = "pre-industrial 1870-1900"
+
+        climStartYear = config.getint('oceanObservations',
+                                      'sstClimatologyStartYear')
+        climEndYear = config.getint('oceanObservations',
+                                    'sstClimatologyEndYear')
+        timeStart = datetime.datetime(year=climStartYear, month=1, day=1)
+        timeEnd = datetime.datetime(year=climEndYear, month=12, day=31)
+
+        if climStartYear < 1925:
+            period = 'pre-industrial'
         else:
-            timeStart = datetime.datetime(1990, 1, 1)
-            timeEnd = datetime.datetime(2011, 12, 31)
-            preindustrialText = "present-day 1990-2011"
+            period = 'present-day'
 
         dsTimeSlice = dsObs.sel(time=slice(timeStart, timeEnd))
         monthlyClimatology = dsTimeSlice.groupby('time.month').mean('time')
 
-        # Rename the observation data for code compactness
         dsObs = monthlyClimatology.transpose('month', 'lon', 'lat')
         obsFieldName = 'SST'
 
         # Set appropriate figure labels for SST
         observationTitleLabel = \
-            "Observations (Hadley/OI, {})".format(preindustrialText)
+            "Observations (Hadley/OI, {} {:04d}-{:04d})".format(period,
+                                                                climStartYear,
+                                                                climEndYear)
```

Comment:
I am ambivalent as to how the observations are preprocessed, but agree with @xylar that this is outside the scope of this PR.  Certainly not doing the time averaging every time is more efficient, but I'm not convinced it will be a large cost savings.  Note the SSS observations also require time averaging.  If the desire of everyone is to preprocess, I'm happy to take this on.